components/stepper/overview.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Overview
+page_title: Stepper Overview
+description: Overview of the Stepper for Blazor.
+slug: stepper-overview
+tags: telerik,blazor,stepper,overview
+published: True
+position: 0
+---
+
+
+# Stepper Overview
+
+The <strong>Stepper for Blazor</strong> is a component that renders a path formed by a sequence of logical steps towards a bigger action. The Stepper indicates the user’s progress within this action by showing the steps left for them to complete it. It makes the user experience less overwhelming as it breaks the long process into smaller parts.
+
+The Stepper provides several features out of the box, including the ability to define the appearance of each step, validation for the steps, strict linear flow and step template to make the it fit any design requirements.
```

Comment:
add links to relevant articles in this paragraph

Reply:
Done